idemp-rest/WEB-INF/src/org/bandahealth/idempiere/rest/service/impl/ChargeTypeRestService.java

```diff
@@ -0,0 +1,50 @@
+package org.bandahealth.idempiere.rest.service.impl;
+
+import javax.ws.rs.Consumes;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.MediaType;
+
+import org.bandahealth.idempiere.rest.IRestConfigs;
+import org.bandahealth.idempiere.rest.model.BaseListResponse;
+import org.bandahealth.idempiere.rest.model.ChargeType;
+import org.bandahealth.idempiere.rest.service.BaseEntityRestService;
+import org.bandahealth.idempiere.rest.service.db.ChargeTypeDBService;
+import org.springframework.beans.factory.annotation.Autowired;
+
+@Path(IRestConfigs.CHARGE_TYPES_PATH)
+@Consumes(MediaType.APPLICATION_JSON)
+@Produces(MediaType.APPLICATION_JSON)
+public class ChargeTypeRestService extends BaseEntityRestService<ChargeType> {
+	@Autowired
+	private ChargeTypeDBService dbService;
+
+	@GET
+	@Path(IRestConfigs.ROOT_PATH)
+	@Override
+	public BaseListResponse<ChargeType> getAll(@QueryParam(IRestConfigs.QUERY_PARAMETER_PAGE) int page,
```

Comment:
With the above code change, you can remove each of these methods.

Reply:
ok